src/pages/admin/adminReports/AdminReports.tsx

```
@@ -1,3 +1,120 @@
+            <S.Item key={data.reportId} to={`${data.reportId}`}>
+              <S.ProfileImg>
+                <Avatar
+                  image={data.profileImg ? data.profileImg : defaultImg}
+                  size='50px'
+                />
+                <S.NickName>{data.nickname}</S.NickName>
+              </S.ProfileImg>
+              <S.ContentArea>
+                {data.category.map((category) => (
+                  <S.Category>
+                    "{REPORT_CATEGORY_LIST[category - 1]}"
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)